src/test/features/response/form/models/whenDidYouPay.ts

```diff
@@ -0,0 +1,147 @@
+/* Allow chai assertions which don't end in a function call, e.g. expect(thing).to.be.undefined */
+/* tslint:disable:no-unused-expression */
+
+import { expect } from 'chai'
+import * as randomstring from 'randomstring'
+import { Validator } from 'class-validator'
+import { expectNumberOfValidationErrors, expectValidationError } from '../../../../app/forms/models/validationUtils'
+import { WhenDidYouPay, ValidationErrors } from 'response/form/models/whenDidYouPay'
+import { LocalDate } from 'forms/models/localDate'
+import * as moment from 'moment'
+import { ValidationConstraints } from 'forms/validation/validationConstraints'
+import { ValidationErrors as DefaultValidationErrors } from 'forms/validation/validationErrors'
+
+describe('WhenDidYouPay', () => {
+  describe('constructor', () => {
```

Comment:
We would like to stop testing constructor as that a bit like testing getters. Please remove.